### PR TITLE
fix: tag former team members as such in blog

### PR DESCRIPTION
**Pull request by nunopolonia**
Fixes SIGN-584

- Add (former) to person component
- Pass is_active to person as prop in blog-entry
- Pass is_active to person as prop in blog-post

### CODE REVIEW

**Comment by nunopolonia** [comment]
Depends on https://github.com/significa/significa.co/pull/297 to be able to pass the CI/CD Github actions

File: src/components/pages/blog-post.svelte [hunk 1]
```diff
@@ -75,7 +75,12 @@
   {#if story.content.author?.id}
     {@const author = story.content.author}
     <div use:drawerLinks class="mx-auto mt-10 max-w-2xl border-b border-t py-8 md:mt-14 lg:mt-20">
-      <Person name={author.name} position={author.content.position} photo={author.content.photo} />
+      <Person
+        is_active={author.content.is_active}
```
Comment (kaaps) [inline src/components/pages/blog-post.svelte:79]: Please change the prop to camelCase ìsActive` is the standard that we use

File: src/components/blog-entry.svelte [hunk 2]
```diff
@@ -22,6 +22,7 @@
   <div class="container mx-auto flex gap-6 px-container py-8 @container">
     <div class="hidden flex-1 @6xl:block">
       <Person
+        is_active={post.content.author.content.is_active}
```
Comment (kaaps) [inline src/components/blog-entry.svelte:25]: camelCase prop

**Comment by nunopolonia** [comment]
@kaaps While doing the changes you requested I've noticed that the Person component is also being used in the testimonials so I've set the isActive prop as true by default, otherwise all testimonials would have "(Former)" attached to them

**Review by kaaps — APPROVED** [review]

**Pull request merged**